synapse/handlers/appservice.py

```diff
@@ -252,7 +262,7 @@ async def _notify_interested_services_ephemeral(
         new_token: int,
         users: Collection[Union[str, UserID]],
     ) -> None:
-        logger.debug("Checking interested services for %s" % (stream_key))
+        logger.debug("Checking interested services for %s" % stream_key)
```

Comment:
I think you wanted this?

```suggestion
        logger.debug("Checking interested services for %s", stream_key)
```

Reply:
Oh! Yes, that's a better shout.